src/node_crypto.cc

```diff
@@ -1256,6 +1316,7 @@ void SecureContext::SetTicketKeys(const FunctionCallbackInfo<Value>& args) {
   ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
   Environment* env = wrap->env();
 
+  // XXX(sam) Move type and len check to js, and CHECK() in C++.
```

Comment:
nit: should these be `TODO(@sam-github)` statements instead?

Reply:
OK, I'll change them.